### DISCUSSION TITLE
UI wording improvement

### DISCUSSION THREAD

**Discussion by pwn0r** [tnodir/fort#116] [General]
This is just a discussion about some wording in UI, as it can see some improvement for better consistency and readability. 
It is worth to cross check suggestions with a few people as well. 

This app uses "Stop traffic" and "Stop Internet traffic". Would it be better to use more standard wording in regards to firewalls, and rename it to "Block All Traffic" and "Block Internet traffic"?

**Comment by tnodir** [comment]
Fixed in [v3.9.10](https://github.com/tnodir/fort/releases/tag/v3.9.10).

Please create separate issues for other UI wording improvements.
